chefignore

```diff
@@ -0,0 +1,9 @@
+*/.chef
```

Comment:
afaik documentation error at docs.chef.io. They changed it in ~2010 and still need to clean it up…